 webpack.common.js | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/webpack.common.js b/webpack.common.js
index 61fe39483..b838552b1 100644
--- a/webpack.common.js
+++ b/webpack.common.js
@@ -20,15 +20,7 @@ const Assets = [
 ];
 
 const DEV_MODE = process.env.NODE_ENV !== 'production';
-let COMMIT_SHA = '';
-try {
-    COMMIT_SHA = require('child_process')
-        .execSync('git describe --always --dirty')
-        .toString()
-        .trim();
-} catch (err) {
-    console.warn('Failed to get commit sha. Is git installed?', err);
-}
+const COMMIT_SHA = process.env.COMMIT_SHA;
 
 const NODE_MODULES_REGEX = /[\\/]node_modules[\\/]/;
 
-- 
2.46.1
